src/com/ableton/Pyenv.groovy

```diff
@@ -8,6 +8,28 @@ package com.ableton
  * @see com.ableton.VirtualEnv
  */
 class Pyenv implements Serializable {
+  /**
+   * Script context.
+   * <strong>Required value, may not be null!</strong>
+   */
+  Object script
+  /**
+   * Python version, as listed in {@code pyenv install --list}.
```

Comment:
Line 40 says 
https://github.com/Ableton/python-pipeline-utils/blob/3c18137f8ffb40f97e933e8237645ebc68c7e896/src/com/ableton/Pyenv.groovy#L40

Reply:
Fixed in 8a3e15f.